Content.Server/DeltaV/Station/Systems/CaptainStateSystem.cs

```diff
@@ -140,6 +129,25 @@ private void HandleNoCaptain(Entity<CaptainStateComponent?> station, CaptainStat
         }
     }
 
+    /// <summary>
+    /// Tries to send a station announcement if entity belongs to a station. If entity does not have a station throw an error and send global anouncement as backup.
+    /// </summary>
+    private void SendAnnouncement(EntityUid ent, string message)
+    {
+        if (_station.GetOwningStation(ent) != null)
+            _chat.DispatchStationAnnouncement(
+                ent,
+                message,
+                colorOverride: Color.Gold);
+        else
+        {
+            Log.Error($"{ent} does not belong to a station.");
```

Comment:
CaptainStateComponent is only added to stations so theres no way this would happen